internal/container/docker.go

```diff
@@ -26,7 +26,7 @@ import (
 	"time"
 )
 
-var ErrDockerNotFound = errors.New("podman not found in your system, check requirements at https://dochub.mongodb.org/core/atlas-cli-deploy-local-reqs")
+var ErrDockerNotFound = fmt.Errorf("%w: docker not found in your system, check requirements at https://dochub.mongodb.org/core/atlas-cli-deploy-local-reqs", ErrContainerEngineNotFound)
```

Comment:
[q] instead of joining the errors here how about doing it on `container_new_linux.go` and `container_new.go`?

Reply:
What would be the advantage of doing that?
Keep in mind that on some linux repos we'll have both linux/podman

Reply:
That is exactly why, one place to wrap the error, I haven't seen the pattern of fmt.Errorf in a package level yet.

It is normally done on functions.